spacewalk/setup/spacewalk-setup.spec

```diff
@@ -130,6 +126,9 @@ setup tasks, re-installation, and upgrades.
 %setup -q
 
 %build
+%if 0%{?rhel} || 0%{?fedora}
+sed -i 's#/etc/pki/trust/anchors#/etc/pki/ca-trust/source/anchors#' lib/Spacewalk/Setup.pm
```

Comment:
There is also `$DEFAULT_DOC_ROOT = "/srv/www/htdocs"` defined. 

I wonder if it would make more sense to put this difference in the code instead of using `sed` to replace it.
We have in bin/spacewalk-setup already a function is_suse_like() which read os-release.

Reply:
Let me move the logic there.

Reply:
Did you do this already? Then you missed to add the files to this commit. Or do you plan this for the future in a different PR?

Reply:
New PR. For reference #3019